docs/content/posts/2024-07-15-release-1.20.0.md

```diff
@@ -0,0 +1,510 @@
+
+-- leaves the selection of an algorithm up to the connector.
+CREATE TABLE MyTable (uid BIGINT, name STRING) DISTRIBUTED BY (uid) INTO 4 BUCKETS;
+
+-- leaves the number of buckets up  to the connector.
+CREATE TABLE MyTable (uid BIGINT, name STRING) DISTRIBUTED BY (uid);
+
+-- only defines the number of buckets.
+CREATE TABLE MyTable (uid BIGINT, name STRING) DISTRIBUTED INTO 4 BUCKETS;
+```
+
+**More Information**
+* [FLINK-33494](https://issues.apache.org/jira/browse/FLINK-33494)
+* [FLIP-376](https://cwiki.apache.org/confluence/display/FLINK/FLIP-376%3A+Add+DISTRIBUTED+BY+clause)
+
+# State & Checkpoint Improvements
+
+## Unified File Merging Mechanism for Checkpoints
+
+The unified file merging mechanism for checkpointing is introduced to Flink 1.20 as an MVP ("minimum viable product") feature,
+which allows scattered small checkpoint files to be written into larger files, reducing the number of file creations
+and file deletions and alleviating the pressure of file system metadata management raised by the file flooding problem
+during checkpoints.
+
+The mechanism can be enabled by setting `execution.checkpointing.file-merging.enabled` to `true`. For more advanced options
+and principle behind this feature, please refer to the document of Checkpointing.
+
+**More Information**
+* [FLINK-33494](https://issues.apache.org/jira/browse/FLINK-32070)
+* [FLIP-306](https://cwiki.apache.org/confluence/display/FLINK/FLIP-306%3A+Unified+File+Merging+Mechanism+for+Checkpoints)
+* [Documentation](https://nightlies.apache.org/flink/flink-docs-release-1.20/docs/dev/datastream/fault-tolerance/checkpointing/#unify-file-merging-mechanism-for-checkpoints-experimental)
+
+## Manually Compact Small SST Files
+
+In some cases, the number of files produced by RocksDB state backend grows indefinitely.This might cause task state
+info (TDD and checkpoint ACK) to exceed RPC message size and fail recovery/checkpoint in addition to having lots of small files.
+
+In Flink 1.20, you can manually merge such files in the background using RocksDB API.
+
+**More Information**
+* [FLINK-26050](https://issues.apache.org/jira/browse/FLINK-26050)
+
+## Reorganizing all State & Checkpointing Configuration Options
+
+In Flink 1.20, all the options about state and checkpointing are reorganized and categorized by prefixes as listed below:
+1. `execution.checkpointing`: all configurations associated with checkpointing and savepoint.
+2. `execution.state-recovery`: all configurations pertinent to state recovery.
+3. `state.*`: all configurations related to the state accessing.
+   a. `state.backend.*`: specific options for individual state backends, such as RocksDB.
+   b. `state.changelog`: configurations for the changelog, as outlined in FLIP-158, including the options for the "Durable Short-term Log" (DSTL).
+   c. `state.latency-track`: configurations related to the latency tracking of state access.
```

Comment:
```suggestion
   b. `state.changelog.*`: configurations for the changelog, as outlined in FLIP-158, including the options for the "Durable Short-term Log" (DSTL).
   c. `state.latency-track.*`: configurations related to the latency tracking of state access.
```

How about adding `.*` to all types? Keep the style the same for all 3 types.

Reply:
Makes sense. Thanks for pointing this out